### ISSUE TITLE
Popover hover trigger on mobile allows click through popover

### ISSUE THREAD

**Issue by philipcolejohnson** [valor-software/ng2-bootstrap#5365]
### Bug description or feature request:
On mobile, the hover trigger seems to allow click propagation through the popover. What seems to be happening is that a finger press runs a mouseout event that closes the popover, and then a click event happens immediately afterward at the same coordinates. Since the popover has just been closed, the element beneath gets clicked instead.

### Plunker/StackBlitz that reproduces the issue:
https://stackblitz.com/edit/angular-bdkni8

Use the browser to view the page in mobile. Click the button to open the popover over the link. Click on the popover where the link is underneath and popover will close and then the page will navigate to the link.

### Versions of ngx-bootstrap, Angular, and Bootstrap:

ngx-bootstrap: 5.1.2

Angular: 8.0.0

Bootstrap: 4.x

**Comment by olierxleben** [comment]
I would like to have a look into that issue, but I dont understand the error. Is this occuring in a specific browser? I tested with Safari and Firefox and I cannot reproduce the behavior?

**Comment by philipcolejohnson** [comment]
You're right. I see the problem only in Chrome. I tried Firefox in the touch-emulator mode (although I didn't try Safari on my phone) and it seemed fine.

As a workaround, I had to detect mobile devices and remove the hover trigger.

**Comment by olierxleben** [comment]
thats sound weird for 2k19. lol.
Just to be sure, which Chrome version are you on?
